src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2CommercialButtons.tsx

```diff
@@ -326,21 +324,20 @@ const ArtworkSidebar2CommerialButtons: React.FC<ArtworkSidebar2CommercialButtons
         </>
       )}
 
-      <Flex>
-        {artwork.isSold && renderCreateAlertButton()}
-        {artwork.isAcquireable && (
-          <Button
-            width="100%"
-            size="large"
-            loading={isCommitingCreateOrderMutation}
-            onClick={handleCreateOrder}
-          >
-            {t("artworkPage.sidebar.commercialButtons.buyNow")}
-          </Button>
-        )}
-        {artwork.isOfferable && (
-          <>
-            <Spacer ml={artwork.isAcquireable ? 1 : 0} />
+      <Flex flexDirection={["column", "column", "column", "column", "row"]}>
+        <Join separator={<Spacer ml={[0, 0, 0, 0, 1]} mt={[1, 1, 1, 1, 0]} />}>
```

Comment:
And actually just simply `<Spacer mt={1} ml={1} />` would work too! 😛

Reply:
thank you!